modules/collections/client/config/collections.routes.js

```diff
@@ -0,0 +1,751 @@
+'use strict';
+
+angular.module('collections').config(['$stateProvider', function($stateProvider) {
+	$stateProvider
+
+	.state('p.collection', {
+		abstract: true,
+		url: '/collection',
+		template: '<ui-view></ui-view>',
+		resolve: {
+			collections: function($stateParams, CollectionModel, project) {
+				return CollectionModel.lookupProject(project.code);
+			},
+			types: function() {
+				var types = ['Inspection Report', 'Certificate', 'Certificate Amendment', 'Management Plan', 'Proponent Self Report'];
```

Comment:
These should be injected as constants in constants.js file.  See core/client/app/constants.js